amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/NoOpWorkerStateChangeListener.java

```diff
@@ -27,4 +27,9 @@ public NoOpWorkerStateChangeListener() {
 	public void onWorkerStateChange(WorkerState newState) {
 
 	}
+
+	@Override
+	public void onInitializationError(Exception e) {
```

Comment:
This should be a default on the interface.
Would prefer the name to be onInitializationException